### DISCUSSION TITLE
Any other Office apps that use Compound file format?

### DISCUSSION THREAD

**Discussion by alanccw** [ironfede/openmcdf#205] [Q&A]
I see https://forensics.wiki/ole_compound_file/ said Office 97-2002 files are using Compound file format.

I check .doc, xls, ppt, and .vsd and find them are in compound format.

Are there any other Office apps that also use this format? I check OneNote and it does not use that format.

**Comment by aunverdorben-dap** [comment]
Outlook .msg files are compound files as well.

**Comment by alanccw** [comment]
@aunverdorben-dap 

Thank you. Any more? Or is there a document on this?

**Comment by Numpsy** [comment]
> Office 97-2002 files are using Compound file format.

I believe that files from newer Office versions can actually be stored as compound documents (rather than the usual zip file) if they are encrypted or protected with RMS (they're still the same office apps though)

**Comment by alanccw** [comment]
@Numpsy 

What is RMS?

I just find Microsoft Project and Publisher are also use this format. But no more.

**Comment by Numpsy** [comment]
> What is RMS?

https://en.wikipedia.org/wiki/Active_Directory_Rights_Management_Services (or maybe the newer Azure / AIP equivalent)

**Comment by alanccw** [comment]
@Numpsy 

After searching online, it seems the RMS does use compound file as it needs some special protection on some streams of the file. I will check that further. Thank you.
